__tests__/api/app.test.js

```diff
@@ -253,4 +253,50 @@ describe('Testing app', () => {
       });
     });
   });
+
+  describe('Endpoint /api/users', () => {
+    describe('Method GET', () => {
+      test('GET: 200 status', () => {
+        return request(app).get('/api/users').expect(200);
+      });
+
+      test('GET: 200 status with all users data and properties', () => {
+        return request(app)
+          .get('/api/users')
+          .then(({ body }) => {
+            const { users } = body;
+            const expectUsers = [
+              {
+                username: 'butter_bridge',
+                name: 'jonny',
+                avatar_url:
+                  'https://www.healthytherapies.com/wp-content/uploads/2016/06/Lime3.jpg',
+              },
+              {
+                username: 'icellusedkars',
+                name: 'sam',
+                avatar_url:
+                  'https://avatars2.githubusercontent.com/u/24604688?s=460&v=4',
+              },
+              {
+                username: 'rogersop',
+                name: 'paul',
+                avatar_url:
+                  'https://avatars2.githubusercontent.com/u/24394918?s=400&v=4',
+              },
+              {
+                username: 'lurker',
+                name: 'do_nothing',
+                avatar_url:
+                  'https://www.golenbock.com/wp-content/uploads/2015/01/placeholder-user.png',
+              },
+            ];
+
+            expect(Array.isArray(users)).toBe(true);
+            expect(users).toHaveLength(4);
+            expect(users).toMatchObject(expectUsers);
+          });
```

Comment:
For this test I would say that if you're going through the effort of writing out the whole array, then just a `.toEqual` would suffice as the _only_ assertion you need to make, because that covers all your other assertions